vllm/model_executor/models/llama.py

```diff
@@ -296,6 +296,12 @@ def sample(
                                    sampling_metadata)
         return next_tokens
 
+    def can_skip_param(self, name: str):
+        # need to skip the names for model (e.g. llama2 70b) as those are not in the param_dict
+        # when loading weights
+        skip_name_list = [".bias", "g_idx", "qweight", "qzeros", "scales"]
```

Comment:
This check seems unnecessary. This should only appear in quantized model. If not quantized, the model would not have these fields. Can you tell us why you added this line?

Reply:
Here is the error we experienced when running the code over the llama-2-70b-chat-hf model, and those names were not in the param_dict list, for example:
```
  File "/opt/conda/envs/py_3.9/lib/python3.9/site-packages/vllm-0.2.7+rocm603-py3.9-linux-x86_64.egg/vllm/model_executor/models/llama.py", line 337, in load_weights
    param = params_dict[name]
KeyError: 'model.layers.0.mlp.down_proj.g_idx'
```

Reply:
You are right about these only appear in quantized model. I was running the test on the quantized version of the model without "quantization" option, and therefore triggered the exception. I reverted that change for now.